### ISSUE TITLE
Did some one already train this mask rcnn on coco dataset and get reasonable results?

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [felixgwu/mask_rcnn_pytorch#6]
I tried the mask rcnn implementation written with tensorflow and I didn't got good results.
Before I try pytorch, I want to ask if some one already get good results after training with COCO dataset.

**Comment by jpchamps** [comment]
any success HuangBo?

**Comment by w102060018w** [comment]
could u share what kind of result u get? I've also try on the tensorflow implementation, but can still only get the bounding box result instead of instance segmentation. If u get the instance segmentation result, is it possible for u to share the modified part of that repo? thanks!
